conf/evolutions/default/9.sql

```diff
@@ -0,0 +1,46 @@
+# core model types
+# --- !Ups
+
+DROP TABLE interest;
+DROP SEQUENCE interest_id_seq;
+
+CREATE SEQUENCE interest_id_seq;
+CREATE TABLE interest (
+  id integer NOT NULL DEFAULT nextval('interest_id_seq'),
```

Comment:
I would not change this, but you can simplify auto_incrementing definitions in Postgres by using the type `colname SERIAL`:
http://www.postgresql.org/docs/9.2/static/datatype-numeric.html#DATATYPE-SERIAL